api/types/plugin.go

```diff
@@ -117,6 +113,23 @@ type PluginConfigInterface struct {
 	Types []PluginInterfaceType `json:"Types"`
 }
 
+// PluginConfigLinux plugin config linux
+// swagger:model PluginConfigLinux
+type PluginConfigLinux struct {
```

Comment:
https://github.com/docker/docker/blob/master/docs/extend/config.md needs to be updated with these details.